### PR TITLE
Test Oracle case insensitive mapping

### PR DESCRIPTION
**Pull request by MiguelWeezardo**
Fixes https://github.com/trinodb/trino/issues/5721 too

### CODE REVIEW

File: plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/OracleQueryRunner.java [hunk 1]
```diff
@@ -36,14 +35,35 @@
 {
     private OracleQueryRunner() {}
 
+    @Deprecated
```
Comment (losipiuk) [inline plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/OracleQueryRunner.java:38]: do we need to keep it around?

Comment (MiguelWeezardo) [reply plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/OracleQueryRunner.java:38]: Not really, I can update those calls.

File: plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestOracleCaseInsensitiveMapping.java [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        assertThat(Stream.of(nameVariants)
+                .map(name -> name.replace("\"", "").toLowerCase(ENGLISH))
+                .collect(toImmutableSet()))
+                .hasSize(1);
+
+        for (int i = 0; i < nameVariants.length; i++) {
+            for (int j = i + 1; j < nameVariants.length; j++) {
+                try (AutoCloseable ignore1 = withTable(TestingOracleServer.TEST_USER + "." + nameVariants[i], "(c varchar(5))");
+                        AutoCloseable ignore2 = withTable(TestingOracleServer.TEST_USER + "." + nameVariants[j], "(d varchar(5))")) {
+                    assertThat(computeActual("SHOW TABLES").getOnlyColumn()).contains("casesensitivename");
```
Comment (losipiuk) [inline plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestOracleCaseInsensitiveMapping.java:148]: This assertion is not needed if you have one below

File: plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestRemarksReportingOracleDistributedQueries.java [hunk 3]
```diff
@@ -29,7 +32,18 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         this.oracleServer = new TestingOracleServer();
-        return OracleQueryRunner.createOracleQueryRunner(oracleServer, ImmutableMap.of(), TpchTable.getTables(), false, true);
+        return OracleQueryRunner.createOracleQueryRunner(
```
Comment (losipiuk) [inline plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestRemarksReportingOracleDistributedQueries.java:35]: changes regarding adding new `createOracleQueryRunner` should be extracted to separate commit.

**Review by losipiuk — COMMENTED** [review]
LGTM module comments

File: plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/OracleQueryRunner.java [hunk 4]
```diff
@@ -88,12 +77,19 @@ public static void main(String[] args)
...
                 ImmutableMap.of("http-server.http.port", "8080"),
-                TpchTable.getTables(),
-                false,
-                false);
+                ImmutableMap.<String, String>builder()
+                        .put("connection-url", server.getJdbcUrl())
+                        .put("connection-user", TEST_USER)
+                        .put("connection-password", TEST_PASS)
+                        .put("allow-drop-table", "true")
+                        .put("oracle.connection-pool.enabled", String.valueOf(false))
```
Comment (losipiuk) [inline plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/OracleQueryRunner.java:89]: "false" (please correct other places too)

**Review by losipiuk — APPROVED** [review]

File: plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java [hunk 5]
```diff
@@ -123,4 +125,14 @@ public void close()
     {
         stop();
     }
+
+    public static Map<String, String> connectionProperties(TestingOracleServer server)
```
Comment (findepi) [inline plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java:129]: That belongs to the query runnner class.

_This_ class is not concerned with Trino connector or its configuration

**Comment by MiguelWeezardo** [comment]
If you think this version is OK, please merge. I don't have the permissions for that.

**Review by eskabetxe — APPROVED** [review]

**Comment by findepi** [comment]
Merged, thanks!
